Dockerfile.in

```diff
@@ -87,4 +87,8 @@ WORKDIR /tmp
 
 # Just dump the binary in the root.
 ADD bin/{ARG_OS}_{ARG_ARCH}/{ARG_BIN} /{ARG_BIN}
+
+# Add License file.
+COPY LICENSE LICENSE
```

Comment:
can you be explicit and say `/LICENSE` or even `/LICENSE.git-sync.txt`

Reply:
I actually need all third-party licenses. I added a make target to generate the license files and added them to the image. see PR: #358